scout/server/blueprints/variants/templates/variants/variant.html

```diff
@@ -489,25 +486,29 @@
       <tbody>
         {% for gene in variant.genes %}
           {% for transcript in gene.primary_transcripts %}
-            <tr {% if transcript.is_primary %} class="bg-info" {% endif %}>
-              <td>
-                <a href="{{ url_for('genes.gene', hgnc_id=gene.hgnc_id) }}">
-                  {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
-                </a>
-                {% if transcript.is_canonical %}
-                  <span class="badge pull-right" title="canonical">C</span>
-                {% endif %}
-              </td>
-              <td>
-                <div>{{ transcript.refseq_id or transcript.transcript_id}}</div>
-              </td>
-              <td title="{{ transcript.coding_sequence_name or '' }}">
-                {{ (transcript.coding_sequence_name or '')|truncate(20, True) }}
-                <span class="text-muted pull-right">
-                  {{ (transcript.protein_sequence_name or '')|url_decode }}
-                </span>
-              </td>
-            </tr>
+            {% for refseq_id in transcript.refseq_identifiers %}
```

Comment:
This table looks a bit too busy. I'd put the summary panel on top of the transcript one by modifying like 53 in this way:
```
<div class="row">
    <div class="col-xs-12">{{ matching_variants() }}</div>
    <div class="col-xs-12 col-sm-6 col-md-4">{{ panel_basics() }}</div>
    <div class="col-xs-12 col-sm-8 col-md-8">
      <div class="col-xs-12 col-sm-12 col-md-12">{{ panel_summary() }}</div>
      <div class="col-xs-12 col-md-12">{{ transcript_overview() }}</div>
    </div>
 </div>
```